corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -284,6 +300,14 @@ class MediaText(XmlObject):
     xpath = NodeField('xpath', TextXPath)
     xpath_function = XPathField('xpath/@function')
 
+    def get_all_xpaths(self):
+        result = {self.xpath_function}
+        if self.xpath:
+            for variable in self.xpath.variables:
+                if variable.xpath:
+                    result.add(str(variable.xpath.function))
```

Comment:
nitpick: why does this need to be cast as a string but not `self.xpath_function` above? It looks like they're both `XPathField`s

Reply:
I don't know if it does. This was copied from the original code and dates back to https://github.com/dimagi/commcare-hq/commit/f96e7876907f31b7331e82831d503b71389665cc

I will remove the `str` and see if there are any issues.